features/flag-on-profile/flag.js

```diff
@@ -0,0 +1,231 @@
+(function() {
+    'use strict';
+    
+    const locationElement = document.querySelector('.location');
+
+    if (locationElement) {
+        const locationText = locationElement.textContent.trim();
+        const countryFlag = getCountryFlag(locationText);
```

Comment:
Just in case the country does not exist, we should add something like this below:
`if (!countryFlag) return;`

Reply:
Good idea